how-to-docs/artifactory-token.md

```diff
@@ -0,0 +1,14 @@
+# Create an Artifactory access token
+
+The requester should state the name of the token, how they would like to recieve the token and post their GPG key.
+
+* Create an [Artifactory][artifactory link] access token using the following command:
+```
+curl -u<username>:<api-key> -XPOST "https://artifactory.digital.homeoffice.gov.uk/artifactory/api/security/token" -d "username=<robot-username>" -d "scope=member-of-groups:<appropriate-groups>" -d "expires_in=9999999999"
+```
+
+where `<robot-username>` is the name of the access token and `<appropriate-groups>` is a comma seperated list of the groups the token should be in.
```

Comment:
I think we should mention appropriate-groups is normally "ci"

Reply:
Will do :+1: